docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -0,0 +1,476 @@
+| `gte`        | Greater than or equal to         |
+| `in`         | Included in an array             |
+| `notIn`      | Not included in an array         |
+| `contains`   | Contains                         |
+| `ncontains`  | Does not contain                 |
+| `containsi`  | Contains, case sensitive         |
+| `ncontainsi` | Does not contain, case sensitive |
+| `null`       | Is null                          |
+| `notNull`    | Is null                          |
+| `between`    | Is between                       |
+| `startsWith` | Starts with                      |
+| `endsWith`   | Ends with                        |
+| `and`        | Logical `and`                    |
+| `or`         | Logical `or`                     |
+
+::: request Example query with filters
+
+```graphql
+{
+  documents(filters: { name: { eq: "test" }, or: [{ price: { gt: 10 }}, { title: { startsWith: "Book" }}] }) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Sorting
+
+Queries can accept a `sort` parameter with the following syntax:
+
+- to sort based on a single value: `sort: "value"` 
+- to sort based on multiple values: `sort: ["value1", "value2"]`
+
+The sorting order can be defined with `:asc` (ascending order, default, can be omitted) or `:desc` (for descending order).
+
+::: request Example request: Sorting on title by ascending order
+
+```graphql
+{
+  documents(sort: "title") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by descending order
+
+```graphql
+{
+  documents(sort: "title:desc") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by ascending order, then on price by descending order
+
+```graphql
+{
+  documents(sort: ["title:asc", "price:desc"]) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Pagination
+
+Queries can accept a `pagination` parameter. Results can be paginated either by page or by offset.
+
+:::note
+Pagination methods can not be mixed. Always use either `page` with `pageSize` **or** `start` with `limit`.
+:::
+
+### Pagination by page
+
+<!-- ? are we using a zero-based or a 1-based index? -->
+
+| Parameter              | Description | Default |
+| ---------------------- | ----------- | ------- |
+| `pagination[page]`     | Page number | 1       |
+| `pagination[pageSize]` | Page size   | 100     |
+
+::: request Example query: Pagination by page
+
+```graphql
+{
+  documents(pagination: { page: 1, pageSize: 10 }) {
+    data {
+      id
+    }
+    meta {
+      pagination {
+        page
+        pageSize
+        pageCount
+        total
+      }
+    }
+  }
+}
+```
+
+:::
+
+### Pagination by offset
+
+| Parameter           | Description                  | Default | Maximum |
+| ------------------- | ---------------------------- | ------- | ------- |
+| `pagination[start]` | Start value                  | 0       | -       |
+| `pagination[limit]` | Number of entities to return | -       | 100     
```

Comment:
The current default is 10 & default maximum is -1 (infinite), same as above, we might tweak those values in the future.